index.html

```diff
@@ -220,6 +222,42 @@ <h3>Multi Post Stories Gain+Glory</h3>
     </div>
   </section>
 
+  <section class="modal-section" id="modal">
+    <div class="modal-container">
+      <div class="modal-content">
+        <div class="modal-image">
+          <div class="times" id="close-btn" onclick>&times;</div>
+          <img id="prj-img" src="images/mobile-popup.png" alt="mobile-popup" class="img" />
+          <img id="prj-img-1" src="images/destkop-popup.png" alt="destkop-popup" class="img" />
+        </div>
+        <div class="modal-title">
+          <h3 class="about-heading modal-heading">Keeping track of hundreds of
+            components</h3>
+        </div>
+        <ul class="modal-list">
+          <li>Ruby</li>
+          <li>CSS</li>
+          <li>JavaScript</li>
+          <li>HTML</li>
+        </ul>
+        <div class="modal-paragraph">
+          <p class="modal-p">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has
+            been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type
+            and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting
+            industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown
+            printer took a galley of type and scrambled it 1960s.</p>
+        </div>
+        <div class="btn">
+          <a href="" class="f-icon-1">See Live <img src="images/external-icon.png" alt="external-icon"></a>
+          <a href="" class="f-icon-2">See Source<img src="images/githubvector.png" alt="github-icon"></a>
+        </div>
+      </div>
+    </div>
+    <div id="overlay"></div>
+  </section>
+
+
+
```

Comment:
- [ ] This is how the project should look like as given in the [design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468). 
<img width="303" alt="Capture" src="https://user-images.githubusercontent.com/58504655/128068470-83bdddb3-ec61-4ab4-a550-ad7178c53d1e.PNG">

-------------------------------------------------------------------------------------------------------------------------

But this is how I can find find it. Please work on it. 
![FireShot Capture 001 - Personal Portfolio - raw githack com](https://user-images.githubusercontent.com/58504655/128068595-cecdbf6a-213c-4d70-b3b4-fb2b01285ba9.png)